src/MovieDetails.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './css/MovieDetails.css'
+
+const MovieDetails = (props) => {
```

Comment:
I appreciate the way you are displaying the movie details once a user clicks. It sticks to the plan on how we envisioned it.